Mathlib/Topology/Algebra/Algebra.lean

```diff
@@ -74,16 +78,471 @@ theorem algebraMapCLM_toLinearMap : (algebraMapCLM R A).toLinearMap = Algebra.li
   rfl
 #align algebra_map_clm_to_linear_map algebraMapCLM_toLinearMap
 
+/-- If `R` is a discrete topological ring, then any topological ring `S` which is an `R`-algebra
+  is also a topological `R`-algebra. -/
+instance DiscreteTopology.continuousSMul [TopologicalSemiring A] [DiscreteTopology R] :
+    ContinuousSMul R A := continuousSMul_of_algebraMap _ _ continuous_of_discreteTopology
```

Comment:
I think this belongs in `Topology.Algebra.MulAction`

Reply:
Yes, but some more work is needed to have it inserted there.
(And probably it can be generalized to the context of Topology.Algebra.MulAction)

Reply:
I tried to put it there, put that requires to add an import (Algebra) to this file, plus inclusion of other lemmas because `ContinuousSMul` follows from the continuity of `algebraMap`.